docs/shared/migration/adding-to-existing-project.md

```diff
@@ -322,21 +302,30 @@ To connect to Nx Cloud:
 - Commit and push your changes
 - Go to [https://cloud.nx.app](https://cloud.nx.app), create an account, and connect your repository
 
-#### Connect to Nx Cloud Manually
-
-If you are not able to connect via the automated process at [https://cloud.nx.app](https://cloud.nx.app), you can connect your workspace manually by running:
-
 ```shell
 npx nx connect
 ```
 
-You will then need to merge your changes and connect to your workspace on [https://cloud.nx.app](https://cloud.nx.app).
+When you are prompted to choose your CI provider, choose `GitHub Actions`.
+
+Once the script is finished, it will print a link in the terminal to register your repository in your [Nx Cloud](https://cloud.nx.app) account. Click the link and follow the steps provided to make sure Nx Cloud is enabled on the main branch of your repository.
+
+### Configure Your CI Workflow
+
+The `nx connect` command you ran in the previous step created a `.github/workflows/ci.yml` file that contains a CI pipeline that will run the `lint`, `test`, `build` and `e2e` tasks for projects that are affected by any given PR.
+
+The key lines in the CI pipeline are:
+
+```yml {% fileName=".github/workflows/ci.yml" %}
+- run: npx nx affected -t lint test build
+- run: npx nx affected -t e2e-ci --parallel 1
```

Comment:
In this case, since we don't have e2e tests, the generator does not add the `e2e-ci` task anymore. We might want to show the full workflow file and highlight the line(s) for the affected commands -- we can snip parts with `# ...` -- just to provide more context around the lines to make them easier to locate.

(same for other standalone tutorials)